design/ui/index.js

```diff
@@ -1 +1,2 @@
-export {EspButton} from './esp-button/dist';
```

Comment:
@lordpixel I am glad you saw this. I am not sure if this file will be manually updated in the long term... I suspect we can auto generate it at some point if it is needed.

This file is consumed by the FramerX tooling that generates all of the code components that Framer will use. Probably makes sense for us to manually add these exports to this file for now.

Reply:
I am going through and resolving all conversations that are NOT part of requested updates.